video/out/opengl/context_drm_egl.c

```diff
@@ -526,7 +528,11 @@ static void queue_flip(struct ra_ctx *ctx, struct gbm_frame *frame)
     data->waiting_for_flip = &p->waiting_for_flip;
     data->log = ctx->log;
 
+
+
     if (atomic_ctx) {
+        if(ctx->vo->opts->drm_opts->drm_send_hdr_meta)
```

Comment:
you definitely want to log something if atomic isn't supported and someone tries to enable this